### PR TITLE
Changes to get SDK using UserAgent and Plug-In Version

### PR DESCRIPTION
**Pull request by MiguelUrrutiaSoler**
Modifications to use the last SDK version 3.3.2 so connect to SDK using UserAgent ECLIPSE_INTEGRATION and the plug-in version
The plug-in version was incremented also.

### CODE REVIEW

**Review by cgperez — APPROVED** [review]

**Review by arturoespinoza15 — APPROVED** [review]

File: features/com.contrastsecurity.ide.eclipse.feature/pom.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
 	<parent>
 		<groupId>com.contrastsecurity.ide.eclipse</groupId>
 		<artifactId>features</artifactId>
-		<version>3.0.5-SNAPSHOT</version>
+		<version>3.0.6-SNAPSHOT</version>
```
Comment (bryanateoan) [inline features/com.contrastsecurity.ide.eclipse.feature/pom.xml:19]: You don't have to update the versions. We can bump the version when we release a new version.

**Review by bryanateoan — CHANGES_REQUESTED** [review]
revert version changes

File: features/com.contrastsecurity.ide.eclipse.feature/pom.xml (hunk 1 above)
Comment (MiguelUrrutiaSoler) [reply features/com.contrastsecurity.ide.eclipse.feature/pom.xml:19]: Ok, because I didn't see a pipeline I thought that was manually updated

Comment (MiguelUrrutiaSoler) [reply features/com.contrastsecurity.ide.eclipse.feature/pom.xml]: Changes reverted

**Pull request merged**